requirements_dev.txt

```diff
@@ -15,7 +15,7 @@ mccabe==0.6.1
 monotonic==1.3
 msgpack-python==0.4.8
 nose==1.3.7
-numcodecs==0.2.1
+numcodecs==0.4.1
```

Comment:
Should we bump the requirement in `setup.py` as well?

Reply:
Yes, good catch, done.